modules/main/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM felixgao/jre8-jacoco:8u92-0.7.6
+
+ENV WASABI_HOME /usr/local/jabba
```

Comment:
Contains jabba in env variables!

Please take a look at our current Dockerfile and [feature/windows](https://github.com/intuit/wasabi/blob/1b83047b7b3c81aa3de313521b2b0de24ab6f9f7/modules/main/src/main/env/docker/wasabi/Dockerfile), maybe we can find a nice middleground?